airbyte-protocol/models/build.gradle

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
     implementation 'javax.validation:validation-api:1.1.0.Final'
     implementation 'org.apache.commons:commons-lang3:3.11'
+
+    implementation project(':airbyte-commons')
```

Comment:
this shouldn't be necessary. this configuration in the `build.gradle` in the root of the project should automatically add commons to every submodule automatically without you having to do it manaully.

https://github.com/airbytehq/airbyte/blob/master/build.gradle#L187-L190